modules/helfi_node_announcement/modules/helfi_global_announcement/helfi_global_announcement.info.yml

```diff
@@ -0,0 +1,12 @@
+name: 'HELfi Global announcement'
+type: module
+core_version_requirement: '^9 || ^10'
+dependencies:
+  - allowed_formats:allowed_formats
+  - drupal:menu_ui
+  - helfi_base_content:helfi_base_content
+  - helfi_node_landing_page:helfi_node_landing_page
```

Comment:
Do we need node landing page and node page dependencies?

Reply:
only node_announcement & external entities is needed